src/components/References/ReferencesEditor.tsx

```diff
@@ -0,0 +1,49 @@
+/*!
```

Comment:
In the body editor we wrap ReferencesEditor with some additional data handling and preparations in the [src/views/ReferencesEditorWrapper.tsx](https://github.com/Atypon-OpenSource/manuscripts-body-editor/pull/167/files#diff-b7aee7258dc899cd6560fc9df7f40ebe0a6785db599425193d2ee3c6f1de3fb4)
I wonder if we can't flatten it and avoid having this file that looks like just adds some opening state handling.